syntax="proto3";
// 指定模板类的包路径
option java_package = "com.szlongzy.common.protobuf.dto";
// 指定模板类的名称，名称必须是有实际业务意义的
option java_outer_classname = "UnbindUserVo";


/* 解绑请求用于【2.3】、【2.5】*/
message unbindRequest{

  // 实际目标设备
  string  objDevId = 1;
  //备用
  string authKey =2;
  string uid = 5;
}

/* 解绑回复*/
message unbindResponse{
  // 回复
  bool  result = 1;
  // 实际目标设备
  string  objDevId = 2;
  // 错误信息
  string  msg = 3;
  //用户
  string uid  =4;

}